pkg/distributor/distributor_test.go

```diff
@@ -655,16 +748,18 @@ func prepare(t *testing.T, numIngesters, happyIngesters int, queryDelay time.Dur
 		limits = &validation.Limits{}
 		flagext.DefaultValues(limits)
 	}
-	limits.IngestionRate = 20
-	limits.IngestionBurstSize = 20
 	cfg.ingesterClientFactory = factory
 	cfg.ShardByAllLabels = shardByAllLabels
 	cfg.ExtraQueryDelay = 50 * time.Millisecond
+	cfg.DistributorRing.HeartbeatPeriod = 100 * time.Millisecond
+	cfg.DistributorRing.InstanceID = strconv.Itoa(rand.Int())
+	cfg.DistributorRing.KVStore.Mock = kvStore
+	cfg.DistributorRing.InstanceInterfaceNames = []string{"eth0", "en0", "lo0"}
```

Comment:
`TestDistributor_PushIngestionRateLimiter` fails in my local machine because none of these interfaces match :/ I will open a PR to get it from the system so that tests pass on all machines.

Reply:
https://github.com/cortexproject/cortex/pull/1915